NuRadioReco/modules/phasedarray/triggerSimulator.py

```diff
@@ -564,11 +564,11 @@ def run(self, evt, station, det,
                                       primary_angles=phasing_angles, trigger_delays=trigger_delays)
 
         trigger.set_triggered(is_triggered)
-
+        
         if is_triggered:
-            trigger.set_trigger_time(trigger_time)
+            #trigger_time(s)= time(s) from start of trace + start time of trace with respect to moment of first interaction = trigger time from moment of first interaction; time offset to interaction time (channel_trace_start_time) already recognized in self.phased_trigger
+            trigger.set_trigger_time(trigger_time)# 
             trigger.set_trigger_times(trigger_times)
-        else:
```

Comment:
I overlooked that before, but you accidentially deleted this line. I.e. all trigger times are now set to Nan